backend/test/card_game/effects_test.clj

```diff
@@ -1,33 +1,48 @@
 (ns card-game.effects-test
   (:require [expectations.clojure.test :refer :all]
-            [card-game.core :as core]))
+            [card-game.core :as core]
+            [configs :as configs]))
 
 (defexpect card-altering
   ; Altering a card in hand works
   (expect
-    [15 10 10 10 10 10 10 10 10 10 10 15]
+    (concat [9001] (subvec (vec (map :power (configs/ini-hand))) 2))
```

Comment:
I don't understand why we play a card at all in this test if we only need to check that altering a card works